benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -212,10 +212,13 @@ def get_all_random_sampling_operators():
     # Get all mxnet operators
     mx_operators = _get_all_mxnet_operators()
 
+    # Deprecated random sampling operators
```

Comment:
Let us not add deprecated operators? 
May be you can update _get_all_mxnet_operators to not return deprecated ops. What do you think?

Reply:
Cool. I'll do that. I thought deprecated ops are still supported (until 2.0) Hence thought of including them. But ya makes sense.